test/exptest.c

```diff
@@ -144,17 +144,22 @@ static int test_mod_exp(int round)
         || !TEST_ptr(m = BN_new()))
         goto err;
 
-    RAND_bytes(&c, 1);
+    if (!TEST_true(RAND_bytes(&c, 1)))
+        goto err;
     c = (c % BN_BITS) - BN_BITS2;
     BN_rand(a, NUM_BITS + c, BN_RAND_TOP_ONE, BN_RAND_BOTTOM_ANY);
```

Comment:
I suppose this BN_rand() call and the one below should be checked as well.

Reply:
Same here.